db/migrate/20240118135350_remove_credits_auto_refreshed_from_organizations.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class RemoveCreditsAutoRefreshedFromOrganizations < ActiveRecord::Migration[7.0]
+  def change
+    remove_column :organizations, :credits_auto_refreshed, :boolean
+
+    add_column :wallets, :upcoming_balance_cents, :bigint, default: 0, null: false
```

Comment:
Rails/BulkChangeTable: You can use `change_table :wallets, bulk: true` to combine alter queries. (https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-change_table, https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/Table.html)